client/app/billing/payment/method/translations/Messages_fr_FR.json

```diff
@@ -17,10 +17,12 @@
   "billing_payment_method_table_action_description_edit": "Modifier la description",
   "billing_payment_method_table_action_set_default": "Définir ce moyen de paiement par défaut",
   "billing_payment_method_table_action_delete": "Supprimer ce moyen de paiement",
-  "billing_payment_method_edit_error": "Une erreur est survenue lors de la modification de la description :",
+  "billing_payment_method_edit_error": "Une erreur est survenue lors de la modification de la description. {{ errorMessage }}",
```

Comment:
```suggestion
  "billing_payment_method_edit_error": "Une erreur est survenue lors de la modification de la description: {{ errorMessage }}.",
```

Reply:
Don't know if there is a final dot to the errorMessage or not. So I don't put them in translations ;-)